watertap/unit_models/mvc/tests/test_mvc.py

```diff
@@ -52,14 +54,16 @@ def build(m):
     # Compressor
     m.fs.compressor = Compressor(default={"property_package": m.fs.properties_vapor})
 
+    # Condenser
+    m.fs.condenser = Condenser(default={"property_package": m.fs.properties_vapor})
```

Comment:
Similarly, does IDAES not have a Condenser unit model available? Just want to refresh my memory on why it was necessary to make custom models for these.

Reply:
This condenser model was written to be used with the water property package and requires that the outlet is fully condensed.